src/client/components/DataGrid/DataCell/DataCell.scss

```diff
@@ -38,6 +38,28 @@ $border: 1px solid $ui-border;
   &.lastRow {
     border-bottom: $border;
   }
+
+  &.highlighted {
+    &:not(.noBorder) {
```

Comment:
This prevents titles like this one to be highlighted: 
![image](https://github.com/openforis/fra-platform/assets/41337901/91807091-5946-40c1-9de5-342ea326d19f)


Alternatively we can pass another flag `highlightEnabled` , like `<DataRow actions={actions} highlightEnabled={false} >` and handle that in the component itself.

Or do we want to highlight those titles as well?